background.js

```diff
@@ -75,6 +75,9 @@ function constructURLs(url){
     if (url.indexOf('youtube.com') != -1) {
         urls = urls.concat(getYoutubeURLs(url));
     }
+    if (url.indexOf('https') != -1) {
```

Comment:
Use [startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) instead of indexOf.